README.md

```diff
@@ -7,70 +7,65 @@ debt in the JS ecosystem.
-embedded in our system, far below the packages we knowingly install day by day.
+A PR has been created and waits to be merged. Nothing to do here for now.
 
-To make things easier, we will be splitting up the work into much smaller
-and more manageable tasks to gradually chip away instead of a big bang solution.
+<img src="https://img.shields.io/badge/pr%20needs%20update-bfd4f2?style=flat" alt="pr needs update" style="vertical-align: middle"/>
 
-Some examples of what we will be doing:
+The PR needs to be updated. Either there was a reponse / review by the maintainer or the PR has merge conflicts
 
-- Remove these packages from popular packages as direct dependencies
-- Fork packages which have officially stated they will never move off these
-packages
-    - Where possible, make these forks track the original source repo (i.e.
-catch up from main when the origin changes).
-- Replace redundant polyfills with native functionality in popular packages
-- Replace these packages with cleaner alternatives
+<img src="https://img.shields.io/badge/blocked-1d76db?style=flat" alt="blocked" style="vertical-align: middle"/>
 
-The main goal is to _contribute_ to the packages we actually use (which depend
-on these things), and as a final resort, fork what is immovable.
+This PR is blocked by some other issue. Check regularly if that issue is resolved
 
-## Contributing
+<img src="https://img.shields.io/badge/rejected-f9d0c4?style=flat" alt="rejected" style="vertical-align: middle"/>
 
-If you'd like to contribute towards the effort, simply find an issue you would
-like to help with and just let us know with a reply that you are going to tackle
-it. We will be happy to help you along the way.
+The PR or the idea of a PR has been rejected by the maintainer. Maybe try again after a few months
 
-A good place to start is the [guide](./docs/guide.md).
+<img src="https://img.shields.io/badge/needs%20alternative-F32096?style=flat" alt="needs alternative" style="vertical-align: middle"/>
+
+The package in question needs a replacement that we can bring forward. Feel free to create one. Maitenance burden is on you!
+
+## Where do I start
+
+- If you like to reach out, look for issues with the <a href="https://github.com/es-tooling/ecosystem-cleanup/issues?q=is%3Aissue+is%3Aopen+label%3A%22needs+first+contact%22"><img src="https://img.shields.io/badge/needs%20first%20contact-b60205?style=flat" alt="needs first contact" style="vertical-align: middle"/></a> label and create an issue (or use some other communication channel). Paste the issue you created as answer to tagged issue here
+- If you want to create PRs, look for the <a href="https://github.com/es-tooling/ecosystem-cleanup/issues?q=is%3Aissue+is%3Aopen+label%3A%22accepts+prs%22"><img src="https://img.shields.io/badge/accepts%20prs-DBE1EF?style=flat" alt="accepts prs" style="vertical-align: middle"/></a> label. Paste the PR link in the tagged issue.
+- If you want to own the whole chain, go through:
```

Comment:
It feels a bit intimidating that there's four steps. I wonder if we could instead provide some general guidance like "For large PRs, you may wish to open an issue in the target repository to ask if the maintainers would be amenable to such a change."

I also think we should mention something like "We generally try to avoid maintainers feeling pressured when creating issues and PRs and thus tackle these issues as individuals rather than mentioning e18e."